cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1582,3 +1582,32 @@ if(webgui)
      INSTALL_COMMAND ""
      SOURCE_DIR ${CMAKE_BINARY_DIR}/ui5/distribution)
 endif()
+
+#------------------------------------------------------------------------------------
+# Check if we need libatomic to use atomic operations in the C++ code. On ARM systems
+# we generally do. First just test if CMake is able to compile a test executable
+# using atomic operations without the help of a library. Only if it can't do we start
+# looking for libatomic for the build.
+#
+include(CheckCXXSourceCompiles)
+check_cxx_source_compiles("
+#include <atomic>
+#include <cstdint>
+int main() {
+   std::atomic<int> a1;
+   int a1val = a1.load();
```

Comment:
Can we add `(void)a1val`? With `-Werror` this will bomb otherwise.

Reply:
Very good point. Should have noticed the warnings in `CMakeFiles/CMakeOutput.log`...